ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -499,7 +506,13 @@ export default function TransactionList({
     setSelectedTransaction(transaction);
   }, []);
 
-  const multichainNetwork = useMultichainSelector(
+  const multichainNetwork = useSelector(
+    getSelectedMultichainNetworkConfiguration,
+  );
+  // We still need this data type which is not compatible with non EVM
+  // testnets because of how the previous multichain network selectors work
+  // TODO: refactor getMultichainAccountUrl to not rely on legacy data types
+  const accountConfigForBlockExplorerLink = useMultichainSelector(
```

Comment:
I think the naming is a bit confusing (given that it's a network config, not an account config), WDYT about using `multicainNetworkForSelectedAccount` instead? (yes, the type is a bit different from the "real network config", but that's ok IMO, given that we'll need to refactor this)